apps/docs-snippets/src/guide/types/vector.test.ts

```diff
@@ -69,9 +68,10 @@ describe(__filename, () => {
     // #import { arrayify, readFile };
 
     const bytecode = await readFile(bytecodePath);
+    const bytecodeAsVecU8 = arrayify(bytecode);
 
     const { value: bytecodeRoot } = await bytecodeContract.functions
-      .compute_bytecode_root(arrayify(bytecode))
+      .compute_bytecode_root(Array.from(bytecodeAsVecU8))
```

Comment:
How come `Array.from()` was necessary now if it wasn't before when using the non-mocked Sway implementation?

Reply:
I forgot to leave a comment regarding this, it seems the encoding has changed with the upgrade, @nedsalk you had some thoughts on this?

Reply:
No special thoughts, I was just mentioning #1950 which enabled the `arrayify(bytecode))` which is being changed here.

Reply:
It seems the current encoding post-upgrade of Forc expects an array now?

Reply:
cc @danielbate any thoughts/insights that you might have?

Reply:
Looks like @Dhaiwat10 in #2018 change to add `UInt8Array` as a viable `Vec` input to the `VecCoder` did not make it's way to the `V1` coder. My b I must've missed it, will push a change now.

Reply:
#2289 should resolve this @maschad.